src/components/asset-list/RecyclerViewTypes.js

```diff
@@ -143,25 +137,6 @@ export const ViewTypes = {
     visibleDuringCoinEdit: false,
   },
 
-  POOLS: {
-    calculateHeight: ({ isOpen, isLast, amountOfRows }) =>
-      isOpen
-        ? TokenFamilyHeaderHeight +
-          (isLast
-            ? ListFooter.height + poolsLastOpenAdditionalHeight
-            : poolsOpenAdditionalHeight) +
-          CoinRowHeight * amountOfRows
-        : TokenFamilyHeaderHeight +
-          (isLast
-            ? ListFooter.height + poolsLastClosedAdditionalHeight
-            : poolsClosedAdditionalHeight),
-    index: 5,
```

Comment:
Just to note: the `index` param for the other view types are updated as part of the Savings PR instead of this one